sdk/eventhub/eventhubs-checkpointstore-blob/src/blobCheckpointStore.ts

```diff
@@ -0,0 +1,254 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Any idea why the diff isnt showing this as a rename and instead shows as a new file?

Reply:
I really don't know. I saw this on my local box too and even went to the trouble of doing the git mv's in a single commit, then doing the changes.